README.md

```diff
@@ -277,7 +277,7 @@ Writing a new type definition is as simple as creating a new package. Start by c
 
 * **main** The entry point to the definition (canonical to "main" in NPM's `package.json`)
 * **browser** A string or map of paths to override when resolving (following the [browser field specification](https://github.com/defunctzombie/package-browser-field-spec))
-* **ambient** Denote that this definition _must_ be installed as ambient
+* **ambient** Denote that this definition should be installed as ambient to the most likely target environment
```

Comment:
Yes, correct. And actually, using the ambient field is going to be becoming _way_ more important with TS 1.8 (see https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#augmenting-globalmodule-scope-from-modules). This actually affects how ambient is used and this field shall live on to warn people.